style.css

```diff
@@ -0,0 +1,307 @@
+body {
+  background-color: #f9f9f9;
+  box-sizing: border-box;
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.navbar {
+  align-items: center;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-around;
+  padding: 4px 0;
+}
+
+.left-nav,
+.mid-nav,
+.right-nav {
+  align-items: center;
+  background: transparent;
+  display: flex;
+}
+
+.left-nav a,
+.mid-nav a,
+.right-nav a {
+  color: #606060;
+  padding: 2px 4px;
+  text-decoration: none;
+}
+
+.left-nav img {
+  max-width: 100%;
+  width: 129px;
+}
+
+.mid-nav {
+  margin: 0 40px;
+}
+
+.mid-nav form {
+  display: flex;
+  max-width: 100%;
+  width: 640px;
+}
+
+form input {
+  border: 1px solid #d3d3d3;
+  box-shadow: inset 0 0 1px grey;
+  box-sizing: border-box;
+  font-size: 16px;
+  font-weight: 500;
+  padding: 6px 4px;
+  width: 80%;
+}
+
+.search-btn {
+  background: #f8f8f8;
+  border: 1px solid #d3d3d3;
+  color: grey;
+  font-weight: 200;
+  width: 10%;
+}
+
+.right-nav ul {
+  display: flex;
+  list-style: none;
+}
+
+.right-nav li {
+  padding: 0 6px;
+}
+
+.right-nav .login-btn {
+  align-items: center;
+  border: 1px solid #0066cd;
+  border-radius: 2px;
+  color: #0066cd;
+  display: flex;
+  font-size: 0.8rem;
+  font-weight: 600;
+  padding: 10px 16px;
+}
+
+.right-nav .login-btn .fa-user {
+  background: #0066cded;
+  border-radius: 50%;
+  color: #fff;
+  font-size: 0.8rem;
+  margin-right: 5px;
+  padding: 8px;
+}
+
+.main {
+  display: flex;
+  margin: 4px auto;
+}
+
+.left {
+  margin: 3rem;
+  width: 50%;
+}
+
+.left .video {
+  height: 360px;
+  width: 660px;
+}
```

Comment:
[OPTIONAL] The Alignment of the video and description section doesn't look like that of original YouTube website. you can change the width to 640px to ensure proper alignment of elements in the main container.